changelog.md

```diff
@@ -158,16 +168,17 @@ Archive button not more used in Tracim. All archive buttons is hide (#2347)
 ### New Features
 
 - Search:
-  - lite search similar to tracim v1
+  - lite search similar to Tracim v1
   - smart search base on ElasticSearch (full text) ([Documentation here](https://github.com/tracim/tracim/blob/develop/backend/doc/setting.md#search-method-using-elastic_search-tracim-23))
 - Easy content move using Drag & Drop
 - Lots of UX improvement
 
 ### New Features (technical)
 
-- It's now possible to configure tracim using environment variables ([Documentation here](https://github.com/tracim/tracim/blob/develop/backend/doc/setting.md#tracim-22-fully-supported-var))
+- It's now possible to configure Tracim using environment variables ([Documentation here](https://github.com/tracim/tracim/blob/develop/backend/doc/setting.md#tracim-22-fully-supported-var))
+If you have used Tracim on Docker before this release, you need to delete this files `tracim_web.ini` `tracim_web.ini` `tracim_caldav.ini` available in this default folder `~/tracim/etc` before starting Tracim 2.3.0 (all files are created on startup)
```

Comment:
This part should be in migration guide part

Reply:
Done